l10n_it_account_stamp/i18n/it.po

```diff
@@ -0,0 +1,109 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* l10n_it_account_stamp
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-12-05 15:25+0000\n"
+"PO-Revision-Date: 2018-12-05 16:31+0100\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"Language: it\n"
+"X-Generator: Poedit 1.8.12\n"
+
+#. module: l10n_it_account_stamp
+#: code:addons/l10n_it_account_stamp/invoice.py:73
+#, python-format
+msgid "Missing account configuration for %s"
+msgstr "Configurazione mancante sui conti di ricavo/costo per il prodotto %s"
+
+#. module: l10n_it_account_stamp
+#: field:product.template,stamp_apply_min_total_base:0
+msgid "Stamp apply min total base"
+msgstr "Base imponibile per l'applicazione"
+
+#. module: l10n_it_account_stamp
+#: view:product.product:0
+msgid "{'readonly': [('is_stamp', '=', True)]}"
+msgstr ""
```

Comment:
Questa deve trovarsi nel po come traducibile? Dove compare?

Reply:
viene inserito in automatico dall'esportazione, non è rilevante